app/services/Athena.scala

```diff
@@ -124,6 +126,35 @@ class Athena() extends StrictLogging {
       .map(_.queryExecutionId())
   }
 
+  private def startVariantViewsQuery(from: String, to: String, channel: String, testName: String): ZIO[ZEnv, AthenaError, QueryExecutionId] = {
+    logger.info(s"Querying athena")
+
+    val query =
+      s"""
+         |SELECT date_hour, ab.variant, COUNT(*) AS views
+         |FROM acquisition.epic_views_prod, UNNEST(abtests) t(ab)
+         |WHERE date_hour >= timestamp '$from' AND date_hour < timestamp '$to'
+         |AND ab.name = '$testName'
```

Comment:
can you do sql injection in Athena? 😀  if I name my AB test `DROP TABLES` what would happen?

Reply:
good question! It can't change the data, but perhaps it can mess with the athena "tables".
It looks like it's possible - https://docs.aws.amazon.com/athena/latest/ug/querying-with-prepared-statements.html
but I'm not sure yet how to do it with the java sdk

Reply:
It's working for the test name now, thanks for suggesting!
See https://github.com/guardian/support-admin-console/pull/606/commits/e96f4e84fcbdff66d530fadbfb065d064220e6af

I can't get it to work with timestamp casting though